protocol/storage/miner.go

```diff
@@ -493,16 +493,19 @@ type dealsAwaitingSealStruct struct {
 	SuccessfulSectors map[uint64]*sectorbuilder.SealedSectorMetadata
 	// Maps from sector id to seal failure error string.
 	FailedSectors map[uint64]string
+	// Maps from sector id to a sectors commitSector message
+	CommitmentMessages map[uint64]cid.Cid
 
 	onSuccess func(dealCid cid.Cid, sector *sectorbuilder.SealedSectorMetadata)
 	onFail    func(dealCid cid.Cid, message string)
 }
 
 func (sm *Miner) loadDealsAwaitingSeal() error {
 	sm.dealsAwaitingSeal = &dealsAwaitingSealStruct{
-		SectorsToDeals:    make(map[uint64][]cid.Cid),
-		SuccessfulSectors: make(map[uint64]*sectorbuilder.SealedSectorMetadata),
-		FailedSectors:     make(map[uint64]string),
+		SectorsToDeals:     make(map[uint64][]cid.Cid),
+		SuccessfulSectors:  make(map[uint64]*sectorbuilder.SealedSectorMetadata),
+		FailedSectors:      make(map[uint64]string),
+		CommitmentMessages: make(map[uint64]cid.Cid),
```

Comment:
This isn't a great to store this. A better solution is coming (see feat/better_testing_for_deals_awaiting_seals), but it comes with a large refactor that is better addressed in a separate PR after this.